### ISSUE TITLE
add davidovski.xyz

### ISSUE THREAD

**Issue by davidovski** [nkoehring/250kb-club#386]
weighs only 226kb as of today

unfortunately large amounts of this size come from the background image, font and other images, but I've compressed these as much as possible without sacrificing entirely on quality. Somehow in previous versions I'd managed to leave a 1.6mb image as the background, in its full resolution with way too much deadspace. Let me know if you think that I can get away with reducing the resolution/quality of anything to help.

**Comment by nkoehring** [comment]
Wow, it is so pretty! Really beautiful page!

The bg image is pretty large... like 2000px wide. Is that necessary? And maybe you can use indexed colours? It looks like it would be fine with an optimized palette of like 64 or so colours. I could even imagine using less colors and dithering could have a nice effect.

Anyway, welcome to the 250kB Club! Your subpage will be https://250kb.club/davidovski-xyz

**Comment by davidovski** [comment]
Thanks a lot, I appreciate it. You make a good point though; I thought that I'd used indexed colours but this didn't carry over when I exported the image as a jpg. (the image only has a 3 color pallet to begin with) Switching to gif format reduced the size to 3/8ths. I could reduce the width further but I think any more would sacrifice on quality more than im willing. 

I've updated it, now the site is 95.66kB total size, much better than before
